modules/snooker/src/main/ts/ephox/snooker/util/CssUtils.ts

```diff
@@ -0,0 +1,21 @@
+import { HTMLElement, HTMLTableElement } from '@ephox/dom-globals';
+import { Arr } from '@ephox/katamari';
+import { SelectorFilter, Attr, Compare, Element } from '@ephox/sugar';
+
+const getPixelWidth = (elm: HTMLElement) => elm.getBoundingClientRect().width;
+const getPixelHeight = (elm: HTMLElement) => elm.getBoundingClientRect().height;
+
+const getIsRoot = (editorBody: Element<any>) => (element: Element<any>) => Compare.eq(element, editorBody);
+
+const removeDataStyle = (table: Element<HTMLTableElement>, styleAttribute: string) => {
```

Comment:
This is very much a TinyMCE concept to have these attributes. I don't think this is well suited for snooker.